task/rhacs-deployment-check/3.71/README.md

```diff
@@ -2,48 +2,101 @@
 
 <https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
 
 ## Install the Task
 
 ```bash
-kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-deployment-check/3.71/raw
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-deployment-check/4.0/raw
 ```
 
 ## Parameters
 
-- **`deployment`**: Filename of deployment manifest. May be relative to workspace root or fully qualified. (example -- kustomize/overlays/dev/deployment.yaml)
-- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`deployment`**: Filename of deployment manifest. May be relative to source workspace root or fully qualified. (example -- kustomize/overlays/dev/deployment.yaml)
 - **`output_format`**:  Examples: _**table**, csv, json, junit_
-- **`rox_central_endpoint`**: Secret containing the address:port tuple for StackRox Central. Default: _**rox-central-endpoint**_
-- **`rox_api_token`**: Secret containing the StackRox API token with CI permissions. Default: _**rox-api-token**_
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central. Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_config_dir`**: Path to the roxctl config directory within the `roxctl-config` workspace (if machine to machine authentication is used). Mutually exclusive with **`rox_token_file`**. The path should be prefixed with `/roxctl-config`. Examples: _"/roxctl-config", **""**_.
+- **`rox_token_file`**: Path to the API Token file (if API Token authentication is used). Mutually exclusive with **`rox_config_dir`**. The path should be prefixed with `/rox-api-token-auth`. Examples: _**""**, "/rox-api-token-auth/rox_api_token"_.
+- `rox_image`: The image providing the roxctl tool (optional). Default: quay.io/stackrox-io/roxctl:4.4.2 (this is also the minimum version working with this task). 
+
+One of the **`rox_config_dir`** or **`rox_token_file`** parameter is required for the authentication against the remote Central to work.
+
 ## Workspaces
 
 - **source**: A [Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md) containing the deployment manifest.
+- **roxctl-config**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces) containing the configuration for roxctl. Used to authenticate with the remote central using short-lived tokens. The content of this worksapce is ideally populated by a rhacs-m2m-authenticate TaskRun. This workspace is mutually exclusive with the `rox-api-token-auth` workspace.
```

Comment:
I wouldn't use Ideally populated but rather that it _has_ to be. While technically possible I don't think we should promote it here.